superset/views/core.py

```diff
@@ -61,7 +61,6 @@
 USER_MISSING_ERR = __("The user seems to have been deleted")
```

Comment:
I'd drop all the changes to this file unless there's a good reason you need them

Reply:
the reason to go to appbuilder.sm.auth_view.login (line 2231) is so that if someone were to use OAuth and define their own security manager it doesn't get stuck in a redirect loop. With default authentication, I believe it works exactly the same.

Reply:
Then that's fine